uat/features/admin_menu.feature

```diff
@@ -23,7 +24,7 @@ Feature: Administration Menu
     And the "Account links" checkbox should be checked
     And the "Shortcuts" checkbox should not be checked
 
-  @api @javascript
+  @api @javascript @deploy @safe
```

Comment:
OK, let's make sure we document it in the README though. We might revisit this decision in the future, but I'm good with declaring a cache clear `@safe`